### PR TITLE
Lesson 1.3

### PR DESCRIPTION
**Pull request by secch97**
- Created <TodoListItem> functional component.
- Handled onSubmit form event.
- Used React States to store a new toDo.
- Refactored code: Renamed handlers and used object destructuring for props.

### CODE REVIEW

File: src/components/TodoListItem.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const TodoListItem = ({toDo}) => {
```
Comment (viniciuspietscher) [inline src/components/TodoListItem.jsx:3]: :100: for destructuring the prop!

File: src/components/AddTodoForm.jsx [hunk 2]
```diff
@@ -1,10 +1,32 @@
...
+        - If todoTitle submitted is blank, then do not change the state.
+        Else, change the state.
+    */
+    if(!todoTitle){
+      return;
+    }
+    console.log(todoTitle);
+    onAddTodo(todoTitle);
+    event.target.reset();
+  };
```
Comment (viniciuspietscher) [inline src/components/AddTodoForm.jsx:24]: Excellent job here! You thought way ahead using trim() and checking if there's no todoTitle and exiting from the function! :1st_place_medal:

**Review by viniciuspietscher — APPROVED** [review]
Great work!!! :1st_place_medal:

File: src/components/TodoListItem.jsx (hunk 1 above)
Comment (secch97) [reply src/components/TodoListItem.jsx:3]: Thank you! I had to read throughout the extra links the book gives us in order to learn object destructuring!

File: src/components/AddTodoForm.jsx (hunk 2 above)
Comment (secch97) [reply src/components/AddTodoForm.jsx:24]: Thank you! I really like validating forms haha!

**Pull request merged**